Core/src/org/sleuthkit/autopsy/datamodel/AbstractAbstractFileNode.java

```diff
@@ -169,10 +169,12 @@ private void removeListeners() {
                     // data sources branch of the tree. The parent nodes in other
                     // branches of the tree (e.g. File Types and Deleted Files) do
                     // not need to be refreshed.
-                    if (parentsChildren instanceof ContentChildren) {
-                        ((ContentChildren) parentsChildren).refreshChildren();
-                        parentsChildren.getNodesCount();
-                    }
+
+// TODO: How will this work with ChildFactory approach?
+//                    if (parentsChildren instanceof ContentChildren) {                     
+//                        ((ContentChildren) parentsChildren).refreshChildren();
+//                        parentsChildren.getNodesCount();
+//                    }
```

Comment:
I'm not sure what to make of this, is this addressed later?

Reply:
Yes, this is addressed later.